package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "ember-bulma",
-  "version": "0.0.0-semantically-released",
+  "name": "@magikcraft/ember-bulma",
+  "version": "1.1.0",
```

Comment:
Also, the version is bumped automatically so no need update this manually.

Reply:
Yeah, this was me publishing a package for my own use.